python-package/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "promptguard"
-version = "0.1.7"
```

Comment:
Seems like there is an occurrence in this following,
docs/reference/library_api.md
3:::: python-package-prompt-guard.src.promptguard.promptguard_service

Reply:
Nice catch!